assets/js/post_office_search.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import AddressSearch from '@18f/identity-address-search';
+
+const root = createRoot(document.getElementById('search'));
```

Comment:
so we have the help center search at the top of every page, I'd recommend renaming this to `address-search` or `post-office-search` just to minimize chances of confusion